plaso/cli/log2timeline_tool.py

```diff
@@ -459,7 +466,18 @@ def ExtractEventsFromSources(self):
     self._SetExtractionPreferredTimeZone(extraction_engine.knowledge_base)
 
     filter_find_specs = None
-    if configuration.filter_file:
+    if configuration.artifacts_filter_file:
```

Comment:
same remark as before, maybe create a helper method for this code and what to do about both filter file and artifacts filter file been set at the same time

Reply:
Moved this to the tools.py so that it can be shared by image_export, log2timeline and psteal.